app/src/main/java/com/amaze/filemanager/adapters/data/OpenFileParcelable.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2014-2020 Arpit Khurana <arpitkh96@gmail.com>, Vishal Nehra <vishalmeham2@gmail.com>,
+ * Emmanuel Messulam<emmanuelbendavid@gmail.com>, Raymond Lai <airwave209gt at gmail.com> and Contributors.
+ *
+ * This file is part of Amaze File Manager.
+ *
+ * Amaze File Manager is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.amaze.filemanager.adapters.data
+
+import android.net.Uri
+import android.os.Parcel
+import android.os.Parcelable
+
+class OpenFileParcelable(
```

Comment:
Use @parcelize for parcelables.

Reply:
kotlin-android-extensions is increasing gradle build time by quite some amount. Skipping it for now.

Reply:
Is this a reported issue?